src/test/java/homeworks/POM/Pages/AllSongsPageHmw.java

```diff
@@ -0,0 +1,27 @@
+package homeworks.POM.Pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+
+public class AllSongsPageHmw extends BasePageHmw{
+    By shufflebuttonLocator= By.cssSelector(".btn-shuffle-all");
+    By firstSongLocator=By.cssSelector("tr.song-item");
+
+
+    public AllSongsPageHmw (WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    public void Shuffle() {
+driver.findElement(shufflebuttonLocator).click();
+    }
+    public void clickFirstSong(){
+        actions.doubleClick(driver.findElement(firstSongLocator));
+
+    }
+
+
+
+}
```

Comment:
**How can I beautify code in IntelliJ:**

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/204494252-8f0b08de-fa8a-420f-b957-52e311281275.png)